src/lib/tasks/server/commit_transaction_task.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include "server_task.hpp"
+
+namespace opossum {
+
+class TransactionContext;
+
+class CommitTransactionTask : public ServerTask<void> {
```

Comment:
Comment? What is this good for and why is it something that's in the network stack?